Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -663,16 +680,16 @@ void Renderer::SetBlendMode(bool forceUpdate)
 bool Renderer::SaveScreenshot(const std::string &filename, const TargetRectangle& rc)
 {
 	if (!s_screenshot_texture)
-		CreateScreenshotTexture(rc);
+		CreateScreenshotTexture();
```

Comment:
This will still crash if you make a screenshot with a small render window first, and afterwards maximizing the render window for the next screenshot.

What do you think about calling CreateScreenshotTexture(rc) unconditionally, which itself check if the size still match? So it might free an old texture, and recreate a new one.